tools/gptfdisk/patches/000-gdisk-only.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Makefile b/Makefile
+index 8c8f9fd..8ea9718 100644
+--- a/Makefile
++++ b/Makefile
+@@ -11,7 +11,7 @@ MBR_LIB_OBJS=$(MBR_LIBS:=.o)
+ LIB_HEADERS=$(LIB_NAMES:=.h)
+ DEPEND= makedepend $(CXXFLAGS)
+ 
+-all:	cgdisk gdisk sgdisk fixparts
```

Comment:
This doesn't matter at all, since ``gdisk`` is host only package.

As I know, ``gdisk`` is intended for interactive use. For scripting and automation, ``sgdisk`` should be used.